developer/src/kmc-ldml/src/compiler/ldml-compiler-messages.ts

```diff
@@ -97,8 +105,8 @@ export class LdmlCompilerMessages {
     m(this.ERROR_DisplayIsRepeated, `display ${LdmlCompilerMessages.outputOrKeyId(o)} has more than one display entry.`);
 
   static ERROR_KeyMissingToGapOrSwitch = SevError | 0x0011;
-  static Error_KeyMissingToGapOrSwitch = (o:{keyId: string}) =>
-  m(this.ERROR_KeyMissingToGapOrSwitch, `key id='${def(o.keyId)}' must have either output=, gap=, or layerId=.`);
+  static Error_KeyMissingToGapOrSwitch = (o:{keyId: string}, x: any) => mx(x,
+    this.ERROR_KeyMissingToGapOrSwitch, `key id='${def(o.keyId)}' must have either output=, gap=, or layerId=.`);
```

Comment:
```suggestion
  static Error_KeyMissingToGapOrSwitch = (o:{keyId: string}, x: any) => mx(x,
    this.ERROR_KeyMissingToGapOrSwitch, 
    `key id='${def(o.keyId)}' must have either output=, gap=, or layerId=.`,
  );
```

format: Only update to this layout for messages that are modified in this PR. Leave existing messages for a separate format-only PR.

I do wonder if `x:any` should be before `o:...` ergonomically?

Reply:
`kmc/build.sh api` (which i've never run locally..) failed in CI!

```
    const event = cls[f]?.({} /* ignore arguments*/);
```

since 'x' is optional… i think i am going to switch these back